Dockerfile

```diff
@@ -1,14 +1,5 @@
-#####################################################################################################
```

Comment:
IIUC you intend to split the konflux dockerfile. Unless the non-konflux dockerfile is used somewhere else (prow CI maybe?) then it can be deleted to avoid unnecessary maintenance for an additional file.

Reply:
The developers use it as part of their workflow for testing changes